test/setup/main.tf

```diff
@@ -67,3 +82,54 @@ module "gke-project-2" {
     "storage-api.googleapis.com",
   ]
 }
+
+/* Create Shared VPC host and service projects to test shared-vpc-helper submodule.
+Only compute.googleapis.com is enabled. All others needed resourcess will provided by the submodule.
+*/
+module "gke_svpc_host_project" {
+  source  = "terraform-google-modules/project-factory/google"
+  version = "~> 3.0"
+
+  name              = "ci-gke-svpc-host"
+  random_project_id = true
+  org_id            = var.org_id
+  folder_id         = google_folder.ci_gke_folder.id
+  billing_account   = var.billing_account
+
+  auto_create_network = false
+
+  activate_apis = [
+    "compute.googleapis.com"
+  ]
+}
+
+// Enables the Shared VPC feature for a created project, assigning it as a Shared VPC host project.
+resource "google_compute_shared_vpc_host_project" "gke_svpc_host_project" {
```

Comment:
This should be handled in the `shared_vpc` example on the `network` module.

Reply:
It cannot be handled in the example in reason example needs already created host and service projects as inputs. But we cannot create svpc service project w/o activated `gke_svpc_host_project`. So we have to use plane resources. Other option, create some dummy-network, just to activate host_project `gke_svpc_host_project`